### PR TITLE
update eth packages versions

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Update Ethereum-related dependencies versions.

## Fixes

Partially addresses #1845 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,11 +28,11 @@
 
 def get_all_extras() -> Dict:
 
-    cosmos_ledger_deps = ["ecdsa==0.15", "bech32==1.2.0"]
+    cosmos_ledger_deps = ["ecdsa>=0.16<1.0", "bech32==1.2.0"]
 
     fetch_ledger_deps = cosmos_ledger_deps
 
-    ethereum_ledger_deps = ["web3==5.12.0", "ipfshttpclient==0.6.1", "eth-account==0.5.2"]
+    ethereum_ledger_deps = ["web3==5.12.0", "ipfshttpclient>=0.6.1,<0.7.0", "eth-account>=0.5.2,<0.6.0"]
```
Comment (DavidMinarsch) [inline setup.py:35]: Can we allow for web3 to be `>=5.12.0, <6.0.0"?

Comment (marcofavorito) [reply setup.py:35]: yes, will try

**Review by solarw — APPROVED** [review]
LGTM

**Comment by DavidMinarsch** [comment]
Closed, as it will largely be irrelevant once we have shipped #2112

**Pull request closed without merging**